src/anomalib/models/image/glass/loss.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2022-2025 Intel Corporation
```

Comment:
For files like these please follow a similar approach https://github.com/open-edge-platform/anomalib/blob/9887e1e6d302486d17419486c237e3e66e377dc0/src/anomalib/models/image/fastflow/torch_model.py#L3